charts/headlamp/values.schema.json

```diff
@@ -35,7 +35,14 @@
       "type": "array",
       "description": "Image pull secrets",
       "items": {
-        "type": "string"
+        "type": "object",
+        "properties": {
+          "name": {
+            "type": "string",
+            "description": "Name of the image pull secret"
+          }
+        },
+        "required": ["name"]
```

Comment:
The docs say the name is not required...

See imagePullSecrets here: https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#containers

If you click [LocalObjectReference](https://kubernetes.io/docs/reference/kubernetes-api/common-definitions/local-object-reference/#LocalObjectReference) from there you see name doesn't say it is required. If a field is required, it always says it's required.